src/Steam.Common/GameId.cs

```diff
@@ -0,0 +1,188 @@
+using System;
+
+namespace Steam
+{
+    /// <summary>
+    /// Definition of a GameId, interpreted from a <see cref="ulong"/> value
+    /// </summary>
+    public readonly struct GameId : IEquatable<GameId>
+    {
+        /// <summary>
+        /// Generic shortcut gameid, contains an empty AppId
```

Comment:
Refer to other types when necessary with normal links. This applies to other docs as well.